src/class-drip-woocommerce-cart-events.php

```diff
@@ -118,7 +122,28 @@ private function remove_drip_cart_session_id()
     private function generate_drip_cart_session_id()
     {
         $random_data = random_bytes(32); // as of php7, random_bytes is advertised as cryptographically secure
-        return hash('sha256',  $random_data); 
+        return hash('sha256',  $random_data);
+    }
+
+    private function find_drip_visitor_uuid()
+    {
+        $account_id = WC_Admin_Settings::get_option( Drip_Woocommerce_Settings::ACCOUNT_ID_KEY );
+        if ($account_id === "") { return; }
```

Comment:
if( empty( $account_id) ) { return; }

that'll also handle $account_id coming back as `false` or `NULL`